### DISCUSSION TITLE
uniapp的上传文件，formData存在name参数，怎么处理？

### DISCUSSION THREAD

**Discussion by PineSongCN** [alovajs/alova#202] [Q&A]
### 请详细描述bug

config参数和formData参数混在了一起，请问怎么处理？
![image](https://github.com/alovajs/alova/assets/39976617/009e479b-5971-4bb7-a2d2-4f744d1257cc)

### 复现步骤

1. 上传文件
2. 文件formData参数存在参数名name

### alova 版本

2.9.0

**Comment by PineSongCN** [comment]
默认的上传文件的方法 uploadFile 只支持post请求，请问是否考虑使用 https://github.com/zlyboy/wx-formdata 封装替换掉uploadFile

**Comment by JOU-amjs** [comment]
问题请在discussion中提问。

就是在post提交的数据里，除了key值为files、file、filePath、name四个值之外的数据，都会作为额外参数自动放到formData里

**Comment by PineSongCN** [comment]
> 问题请在discussion中提问。
> 
> 就是在post提交的数据里，除了key值为files、file、filePath、name四个值之外的数据，都会作为额外参数自动放到formData里

formData参数也需要name字段，和file的name不同，怎么传值能给formData加上name？

**Reply by JOU-amjs** [reply]
这种情况目前可能不支持
